src/main/java/com/github/lunatrius/schematica/client/printer/SchematicPrinter.java

```diff
@@ -332,6 +332,7 @@ private boolean placeBlock(final WorldClient world, final EntityPlayerSP player,
         success = placeBlock(world, player, itemStack, offset, side, hitVec, hand);
         for (int i = 0; success && i < extraClicks; i++) {
             success = placeBlock(world, player, itemStack, offset, side, hitVec, hand);
+            Thread.sleep(50);
```

Comment:
No. This is not even a temporary solution to the real issue and only works for the default settings. Try setting the placement delay to minimum with instant placement enabled.

Whenever you'll move around an area with lots of blocks that require multiple clicks the entire game will freeze for `50 * extra clicks` milliseconds.